packages/cache-map/types.d.ts

```diff
@@ -0,0 +1 @@
+export * from './src/cachemap.js';
```

Comment:
Am I doing this right?

Reply:
I’ve commandeered the branch and aligned this with the new policy.